dotcom-rendering/src/layouts/CommentLayout.tsx

```diff
@@ -313,6 +313,8 @@ export const CommentLayout = ({
 
 	const renderAds = canRenderAds(article);
 
+	const selectedPillar = getCurrentPillar(article);
```

Comment:
Given that we now have `selectedPillar` as a property of `NavType`, are we able to derive this in the `render.article` file as is now the case in `render.front`? A few advantages might be:

- Consistency with the fronts behaviour.
- Deriving it in one place, as opposed to multiple layout files.
- Makes it clear this is the pillar for the nav, to avoid the possibility of it being used in place of `ArticleTheme` by mistake.

Reply:
oh this is a nice idea - yes let me do this